src/Motor.Extensions.Hosting.RabbitMQ/RabbitMQConnectionFactory.cs

```diff
@@ -31,6 +39,8 @@ public interface IRabbitMQConnectionFactory<T> : IDisposable
 
 public sealed class RabbitMQConnectionFactory<TC> : IRabbitMQConnectionFactory<TC>
 {
+    private static readonly X509NameType[] AdditionalNamesToVerify = { X509NameType.DnsName, X509NameType.UpnName };
```

Comment:
Is this actually used anywhere?

Reply:
Nah, good catch! Added that one while I was narrowing down the validation problem and forgot to remove it. Thanks!